src/os2datascanner/projects/static/scss/components/_forms.scss

```diff
@@ -559,6 +560,13 @@
   box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.1);
 }
 
+#file-clear-btn {
+  position: absolute;
+  width: 5%;
+  height: 8.5%;
+  margin-left: 20px;
+}
```

Comment:
Does this conform to [the button guide](https://os2datascanner.readthedocs.io/frontend/buttons/buttons/) in the documentation?

Reply:
Thanks for the docs ! Button is  now fixed and up-to-date with the docs rules !